Makefile

```diff
@@ -118,7 +118,7 @@ dump-test-schema:
 	go run ./tools/dbutils ./server/datastore/mysql/schema.sql
 
 test-go: dump-test-schema
-	go test -tags full,fts5,netgo -parallel 8 -coverprofile=coverage.txt -covermode=atomic ./...
+	go test -tags full,fts5,netgo -parallel 8 -coverprofile=coverage.txt -covermode=atomic ./cmd/... ./ee/... ./orbit/... ./pkg/... ./server/... ./tools/...
```

Comment:
So `./...` somehow didn't catch all of these? Weird.

Reply:
It did catch all of them but there's a new issue with one of our `node_modules` including some Go code that breaks the test build. Sadly, I couldn't find a way to exclude with the `./...` syntax, so I replaced it with the list of directories.

This was originally reported by @mna.

Reply:
Ah right right!